src/main/java/frc/team3388/robot/RobotMap.java

```diff
@@ -7,7 +7,11 @@
 
     // EXAMPLE SYSTEM
     public static final int EXAMPLE_SYSTEM_MOTOR = 5;
-
     // HID
     public static final HidChannel XBOX = RoboRio.newHidChannel(0);
+    public static final int INTAKE_SYSTEM_MOTOR = 1;
+    public static final int PISTON_REVERSE_CHANNEL = 2;
+    public static final int PISTON_FORWARD_CHANNEL = 3;
+    public static final double INTAKE_SPEED = 0.6;
```

Comment:
`RobotMap` is only for connections. This is not.

Put this in the system class.

Reply:
ok